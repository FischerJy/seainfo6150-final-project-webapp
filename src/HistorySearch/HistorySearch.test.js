import React from "react";
import { render } from "@testing-library/react";
import HistorySearch from "./HistorySearch.jsx";
import { BrowserRouter } from "react-router-dom";



it("renders the HistorySearch component correctly", () => {
    const { container } = render(<BrowserRouter>
        <HistorySearch />
    </BrowserRouter>);
    expect(container).toMatchSnapshot();
});