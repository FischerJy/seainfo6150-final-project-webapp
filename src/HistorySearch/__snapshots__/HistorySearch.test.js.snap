// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the HistorySearch component correctly 1`] = `
<div>
  <div
    class="main"
  >
    <br />
    <br />
    <p
      style="text-align: center;"
    >
      name: 
      <input
        type="text"
      />
      <button>
        Search
      </button>
    </p>
  </div>
</div>
`;
